solutions/sum_numbers.py

```diff
@@ -0,0 +1,61 @@
+""" sum numbers 
+sum numbers is a recursive function return the sum of count down numbers 
+
+created: 25/12/2024
+Team number:28
+Team name:MIT Alpha
+The author: Maab Mohamedkhair"""
+
+
+def sum_numbers(n):
+    
+    """ sum numbers 
+sum numbers is a recursive function return the sum of count down numbers 
+from n -> 1
```

Comment:
the function doc string describes the strategy it must only describes the behavior of the function